main/src/library/Fixpoint3/Boxed.flix

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2021 Magnus Madsen
+ * Copyright 2025 Casper Dalgaard Nielsen
+ *                Adam Yasser Tallouzi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+mod Fixpoint3 {
+    import java.lang.Object
+
+    ///
+    /// Represents a boxed value.
+    ///
+    pub enum Boxed {
+        case BoxedBool(Bool)
+        case BoxedChar(Char)
+        case BoxedInt8(Int8)
+        case BoxedInt16(Int16)
+        case BoxedInt32(Int32)
+        case BoxedInt64(Int64)
+        case BoxedFloat32(Float32)
+        case BoxedFloat64(Float64)
+        case None
```

Comment:
Bit surprising to me that there is a none box, maybe this should be explained in the doc text?

Reply:
Calling it None is probably not a good idea.